classes/class-install.php

```diff
@@ -71,9 +71,6 @@ public function __construct( $plugin ) {
 		$this->db_version = $this->get_db_version();
 		$this->stream_url = self_admin_url( $this->plugin->admin->admin_parent_page . '&page=' . $this->plugin->admin->settings_page_slug );
 
-		// Install the plugin.
-		add_action( 'wp_stream_before_db_notices', array( $this, 'check' ) );
-
```

Comment:
@kasparsd Removing this may have broken CI. The tests are still passing locally tho. Any ideas what's going on with Travis-CI?